packages/plugin-ext/src/common/plugin-api-rpc-model.ts

```diff
@@ -15,6 +15,7 @@
 // *****************************************************************************
 
 import * as theia from '@theia/plugin';
+import * as Monaco from '@theia/monaco-editor-core';
```

Comment:
The fact that we use monaco types here a major code smell, IMO. But out of scope.

Reply:
I've adjusted this import to `import type * as Monaco from '@theia/monaco-editor-core'`. That at least guarantees that we're not using any actual Monaco code here; just types.